app/scripts/services/dimLoadoutService.factory.js

```diff
@@ -247,6 +247,10 @@
         }
 
         var items = angular.copy(_.flatten(_.values(loadout.items)));
+        // filter out to only include items for that class
```

Comment:
now it will show "2 of 2 items in your loadout were transferred/equipped" basically the same subclass burns are all treated as one item.